README.md

```diff
@@ -1 +1,48 @@
-# Hello World
+# Github Governance
+
+This readme is for building out our github gonvernance.
+
+## Top level SWRCB GitHub Documentation that covers our guiding principals, protocols, management decisions, and policies
+Also, this is open to discussion. 
+
+### Things that might go in this documnet or in this repo:
+1. Data Governanace guiding pricipals and documents
+	1. A code of conduct [possibily this one](https://github.com/skills/.github/blob/main/CODE_OF_CONDUCT.md)
+	2. Open data policy documentation
+		1. CHHS has a [Open Data Handbook](https://chhsdata.github.io/opendatahandbook/) we might considering forking or adopting in all or part. [Their githubpage](https://github.com/chhsdata/opendatahandbook)
+2. Github account management, repo administration, 
+	1. Default Account permissions for WaterBoard members and outside collaborators
+	2. You can define which individuals or teams own certain types or areas of code in a CODEOWNERS file.
+	3. Other ideas to implement
+		1. Contributing guidlines for internal/external collaborators [possibly this one](https://github.com/skills/.github/blob/main/CONTRIBUTING.md)
+		2. Justification for creation of Repos
+		3. Actions to be taken after the creation of repos.
+2. GitHub Wiki
```

Comment:
another wiki template to consider: https://github.com/18F/Project-Leadership-Collective/wiki/Project-Leadership-Collective

Reply:
another wiki resource to consider: https://github.com/18F/Equity-centered-design-with-American-Indians-and-Alaska-Natives-10x/wiki